### DISCUSSION TITLE
Rotor37 tutorial result

### DISCUSSION THREAD

**Discussion by zznowhere** [mdolab/dafoam#167] [Tutorials]
Hi, 

I am running Rotor37 compressor tutorial (version 2.2.6). It does not seem to converge within 50 iterations. May I have the baseline and the final objective function values TPR, MFR and CMZ to validate my run?

Is version 2.2.8 going to give a different result on this tutorial?

Thanks,
Alice

**Comment by friedenhe** [comment]
Refer to the FAQ on dafoam.github.io. We recently added a new capability to constraint mesh quality during optimization. It will be helpful for turbomachinery cases.

**Reply by zznowhere** [reply]
@friedenhe Thanks for the reply. Have you tried to use different outlet boundary conditions for the Rotor37 example, say specifying mass flow rate with a consistent pressure condition instead of using fixed value for pressure?
